crates/plugins/src/manifest.rs

```diff
@@ -38,6 +39,19 @@ impl PluginManifest {
     pub fn license(&self) -> &str {
         self.license.as_ref()
     }
+
+    pub fn description(&self) -> Option<&str> {
+        self.description.as_deref()
+    }
+
+    pub fn homepage_url(&self) -> Option<Url> {
+        self.homepage
+            .as_deref()
+            .into_iter()
+            .flat_map(Url::parse)
+            .next()
```

Comment:
This might be a bit easier to read:

```suggestion
        Url::parse(&self.homepage?).ok()
```

Reply:
Thanks!